xcoll/scattering_routines/fluka/engine.py

```diff
@@ -157,9 +157,31 @@ def start(cls, *, input_file=None, line=None, elements=None, names=None, cwd=Non
         if this.is_running():
             print("Server already running.", flush=True)
             return
+
+        timeout = 180 # Checking for 3 minutes if _fluka and _flukaserver exist.
+        start_time = time.time()
+
+        while time.time() - start_time < timeout:
+            if this._fluka.exists() and this._flukaserver.exists():
+                break
+            time.sleep(1)
+
         if not this._fluka.exists():
-            raise ValueError(f"Could not find fluka executable {this._fluka}!")
+            print(f"Could not find fluka executable {this._fluka}!")
+            print("Trying modifying the eos path project-f->project/f")
+            fluka_folder = os.path.dirname(this._fluka)
+            if not os.path.exists(fluka_folder):
+                print(f"Could not find the folder for fluka executable {fluka_folder}!")
+            this._fluka = this._fluka.replace("project-f", "project/f")
+            if not this_fluka.exists():
+                raise ValueError(f"Could not find fluka executable {this._fluka}!")
         if not this._flukaserver.exists():
+            flukaserver_folder = os.path.dirname(this._flukaserver)
+            if not os.path.exists(flukaserver_folder):
+                print(f"Could not find the folder for fluka executable {flukaserver_folder}!")
+            this._flukaserver = this._flukaserver.replace("project-f", "project/f")
+            if not this_fluka.exists():
+                raise ValueError(f"Could not find fluka executable {this._flukaserver}!")
             raise ValueError(f"Could not find flukaserver executable {this._flukaserver}!")
```

Comment:
I assume that the above is not entirely correct (there is a `raise ValueError` that will anyway be called once inside the `if`, and there is some checking for `fluka` instead of `flukaserver`)?